### PR TITLE
Vue js

### PR DESCRIPTION
**Pull request by heatherbooker**
@dharness

### CODE REVIEW

File: public/javascripts/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+window.onload = function() {
```
Comment (dharness) [inline public/javascripts/dashboard.js:1]: I don't think you need this window.onload thing..

File: views/index.pug [hunk 2]
```diff
@@ -8,7 +8,7 @@ block content
           h1.cc-logo Chubbo-Chat
       .col-xs-4
         p.cc-loginBtn login
-        p.cc-logoutBtn logout
+        p.cc-logoutBtn(style='display: none') logout
```
Comment (dharness) [inline views/index.pug:11]: You will be fixing this in the next PR, but please dont do this

File: views/layout.pug [hunk 3]
```diff
@@ -3,6 +3,7 @@ html
   head
     title= title
     meta(meta name='viewport' content='width=device-width, initial-scale=1')
+    link(rel='shortcut icon', href='/images/favicon.ico', type='image/x-icon')
```
Comment (dharness) [inline views/layout.pug:6]: I have a different favicon for you

**Comment by dharness** [comment]
LGTM

File: public/javascripts/dashboard.js (hunk 1 above)
Comment (heatherbooker) [inline public/javascripts/dashboard.js:1]: hahahahahahahaha oh but i do....

File: views/layout.pug (hunk 3 above)
Comment (heatherbooker) [inline views/layout.pug:6]: oh, im overzealous over here

**Pull request merged**